package.json

```diff
@@ -67,6 +67,7 @@
 		"@wordpress/element": "2.9.0",
 		"@wordpress/format-library": "1.10.0",
 		"@wordpress/i18n": "3.7.0",
+		"@wordpress/is-shallow-equal": "1.6.1",
```

Comment:
I see there is actively used `react-pure-render/shallowEqual` in Calypso but the package is no longer maintained and Gutenberg offers this alternative should be better.

Reply:
Good spot: #37804